x-pack/plugins/secops/public/components/timeline/footer/index.tsx

```diff
@@ -151,15 +152,29 @@ export class Footer extends React.PureComponent<FooterProps, FooterState> {
   public readonly state = {
     isPopoverOpen: false,
     paginationLoading: false,
+    updatedAt: null,
   };
 
   public componentDidUpdate(prevProps: FooterProps) {
-    const { paginationLoading } = this.state;
-    const { isLoading } = this.props;
+    const { paginationLoading, updatedAt } = this.state;
+    const { isLoading, getUpdatedAt } = this.props;
     if (paginationLoading && prevProps.isLoading && !isLoading) {
       this.setState({
         ...this.state,
         paginationLoading: false,
+        updatedAt: getUpdatedAt(),
+      });
+    }
+    if (updatedAt === null) {
+      this.setState({
+        ...this.state,
+        updatedAt: getUpdatedAt(),
+      });
+    }
+    if (prevProps.isLoading && !isLoading) {
+      this.setState({
+        ...this.state,
+        updatedAt: getUpdatedAt(),
```

Comment:
This file seems weird to me it's using `setState` still? Wondering if we are going to move it to redux anytime soon.

Not asking for changes, just thoughts on it.

Reply:
Let me think more about this one, but it makes sense. I won't change it for this PR.